docs/contributor/06-module-migration-guide.md

```diff
@@ -0,0 +1,231 @@
+# Module Migration Guideline
+
+This guide provides detailed instructions for how to migrate a Kyma module from existing (old) module metadata, especially channel-based ModuleTemplates, to the new module metadata, i.e. version-based ModuleTemplates accompanied by ModuleReleaseMeta.
+
+It is highly recommended that module teams familiarize themselves with the new module metadata before starting the migration:
+- The ADR backing the migration is [#984](https://github.com/kyma-project/community/issues/984)
+- An Update on Module Metadata has also been given in the 2024-11-26 Iteration Review
+
+## Target Process
+
+The target process is shown in the figure below. As an example, the `telemetry` module is used. The promotion of module versions and their assignments to channels is managed via three individual processes. In addition, there is a fourth process to delete a module version.
+
+![Target Process](./assets/module-migration.svg)
```

Comment:
Still, I would work on the diagram a little bit:
- The red lines are too thick, and the text is not very visible
- Two lines in the first process are not touching the target object
- I would change the extension to `drawio.svg` - that allows for editing the diagram in the VSC.

Reply:
The extension is still svg. As for all assets in this folder. You mean you want the drawio infix? Can you elaborate why, since the other svgs in that folder also don't have that infix.

Saving the actual `.drawio` file additionally could make sense. But itself can not be viewed in gh md renderer.